### PR TITLE
Eval

### PR DESCRIPTION
**Pull request by kimberly-glock**

### CODE REVIEW

File: eval_scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+* `test_csv_path = sys.argv[1] # READ IN`
+* `predicted_masks_path = sys.argv[2] # READ IN`
+* `groundtruth_masks_path = sys.argv[3] # READ IN` 
+* `metrics_csv_path = sys.argv[4] # OUTPUT`
+
+Example command line code for the binary task 1 script:
+* `python evaluate-task1-binary.py .\binary\test.csv  .\binary\team-xyz\predictions .\binary\groundtruth .\binary\team-xyz\metrics`
+
+Example command line code for the parts task 2 script:
+* `evaluate-task2-parts.py .\parts\test.csv  .\parts\team-medhacker\predictions .\parts\groundtruth .\parts\team-medhacker\metrics`
```
Comment (anandmalpani) [inline eval_scripts/README.md:13]: `python` is missing here in the command line code.

File: eval_scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Example of folders structure used by the organizers for challenge submissions:
+
+```|__ binary/
+    |
+    |__ test.csv
+    |
+    |__ groundtruth/
+    |   |__ b-qwrnkladsfsa.png
```
Comment (anandmalpani) [inline eval_scripts/README.md:22]: I think this typo may have gotten carried over. The filename should be `b-asdasdsadasd.png`

File: eval_scripts/README.md (hunk 1 above)
Comment (kimberly-glock) [reply eval_scripts/README.md:13]: @anandmalpani not sure what happened there lol. Let me add it back in!

**Review by anandmalpani — COMMENTED** [review]
Some minor changes to be made.

File: eval_scripts/README.md (hunk 2 above)
Comment (kimberly-glock) [reply eval_scripts/README.md]: @anandmalpani done!

**Review by anandmalpani — APPROVED** [review]
Looks good. Let us use the squash and merge. Please add a commit message about the entire commit that is going to the main branch.

**Comment by kimberly-glock** [comment]
micc-189 updated the readme file with information and formatting and added eval scripts 1 & 2

**Pull request merged**